src/doc/src/reference/workspaces.md

```diff
@@ -39,6 +39,9 @@ To create a workspace, you add the `[workspace]` table to a `Cargo.toml`:
 
 At minimum, a workspace has to have a member, either with a root package or as
 a virtual manifest.
+It's also a good practice to specify the `resolver = "2"` unless it is necessary
```

Comment:
This should link out to the documentation for the `resolver` field (see above)

Reply:
done